mongodb.js

```diff
@@ -0,0 +1,72 @@
+// CRUD
+
+const { MongoClient } = require("mongodb");
+
+// eslint-disable-next-line no-multi-spaces
+const { mongoURI } = require("./keys_dev");
+
+// find all in a colletion
+const findAllIn = (colName, query) => {
+  const client = new MongoClient(mongoURI, { useNewUrlParser: true });
```

Comment:
I'm not very experienced in mongodb but I don't think you need to connect for each function. @tawfiknasser have you used this?